interface/usergroup/facilities.php

```diff
@@ -40,6 +40,44 @@
 
 $alertmsg = '';
 
+function refreshCalendar() {
```

Comment:
Would suggest that you take this function and add it to /library/formsoptions.inc.php and call it from there.

Reply:
maybe use calender.inc